AutomatticTracks/build.gradle

```diff
@@ -4,37 +4,48 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.1'
+        classpath 'com.android.tools.build:gradle:4.1.2'
         classpath 'com.novoda:bintray-release:0.9'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.30'
     }
 }
 
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: 'com.novoda.bintray-release'
+apply plugin: 'kotlin-android'
 
 repositories {
     google()
     jcenter()
 }
 
 dependencies {
-    implementation 'io.sentry:sentry-android:1.7.16'
-    implementation 'com.android.support:support-annotations:28.0.0'
+    implementation 'io.sentry:sentry-android:4.1.0'
+    implementation "androidx.annotation:annotation:1.1.0"
+    testImplementation "junit:junit:4.13"
+    testImplementation "org.mockito:mockito-core:3.3.3"
+    testImplementation 'org.robolectric:robolectric:4.4'
+    testImplementation 'androidx.test:core:1.3.0'
+}
+
+project.ext {
+    versionName = "1.2.1"
```

Comment:
I think this is not a great name because it matches the keyword android DSL expects. And.. I don't think it actually works. Currently, `VERSION_NAME` field gets assigned to `null`.

I don't think we need to make it this complicated, the following should work as expected:

```
android {
    compileSdkVersion 30

    defaultConfig {
        versionName "1.2.1"
        minSdkVersion 14

        buildConfigField "String", "VERSION_NAME", "\"$versionName\""
    }

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
}
```

Reply:
Right, thanks for pointing it. It indeed will be null. I'll take care of this change in next PRs.

Reply:
The same as here https://github.com/Automattic/Automattic-Tracks-Android/pull/51#discussion_r587282929 - let's postpone it to AGP `4.1.x` update